lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -100,7 +101,11 @@ def __init__(self, use_authentication=settings.use_auth_http):
     def setup(self):
         return NotImplementedError()
 
-    def _render_error(self, request, failure, version=jsonrpclib.VERSION_1, response_code=FAILURE):
+    def _render_error(self, failure, request, version=jsonrpclib.VERSION_1, response_code=FAILURE,
```

Comment:
http://www.informit.com/articles/article.aspx?p=1392524:
```
Boolean arguments loudly declare that the function does more than one thing. They are confusing and should be eliminated.
```

Its generally better to refactor functions with boolean flags into new functions.  For example

```
def _log_and_render_error(self, failure, request, msg, **kwargs):
    log_support.failure(failure, log, msg)
    self._render_error(failure, request, **kwargs)
```

would be cleaner as it leaves `_render_error` simple, more readable and only doing what its name implies it does.